content/updates/2019-09-25-v12-is-here.md

```diff
@@ -0,0 +1,132 @@
+  * [CVE-2019-15211](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-15211)
+  * [CVE-2019-15212](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-15212)
+  * [CVE-2019-15214](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-15214)
+  * [CVE-2019-15215](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-15215)
+  * [CVE-2019-15216](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-15216)
+  * [CVE-2019-15218](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-15218)
+  * [CVE-2019-15220](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-15220)
+  * [CVE-2019-15221](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-15221)
+  * [CVE-2019-15292](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-15292)
+  * [CVE-2019-3701](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-3701)
+  * [CVE-2019-3819](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-3819)
+  * [CVE-2019-3900](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-3900)
+  * [CVE-2019-9506](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-9506)
+  * [CVE-2019-10638](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-10638)
+  * [CVE-2019-13648](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-13648)
+  * [CVE-2019-14283](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-14283)
+  * [CVE-2019-14284](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-14284)
+  * [CVE-2019-3900](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-3900)
+  * [CVE-2019-15718](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-15718)
+  * [CVE-2018-20406](https://people.canonical.com/~ubuntu-security/cve/CVE-2018-20406)
+  * [CVE-2018-20852](https://people.canonical.com/~ubuntu-security/cve/CVE-2018-20852)
+  * [CVE-2019-10160](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-10160)
+  * [CVE-2019-5010](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-5010)
+  * [CVE-2019-9636](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-9636)
+  * [CVE-2019-9740](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-9740)
+  * [CVE-2019-9947](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-9947)
+  * [CVE-2019-9948](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-9948)
+  * [CVE-2019-5481](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-5481)
+  * [CVE-2019-5482](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-5482)
+  * [CVE-2019-5481](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-5481)
+  * [CVE-2019-5482](https://people.canonical.com/~ubuntu-security/cve/CVE-2019-5482)
+  * [LP-1842651](https://launchpad.net/bugs/1842651)
+  * [LP-1842447](https://launchpad.net/bugs/1842447)
+
+# Support
+
+We are committed to improving the user experience of government. If you have questions, please don't hesitate to reach out at cloud-gov-support@gsa.gov. 
```

Comment:
perhaps link StatusPage? e.g.

` and subscribe to service updates at the [cloud.gov StatusPage](https://cloudgov.statuspage.io/).`